src/Symfony/Component/Validator/CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 CHANGELOG
 =========
 
+5.3
+---
+
+ * Add a `valuesFormat` option to comparison constraints to format datetimes in
```

Comment:
Maybe it is better to list them all explicitly?

Reply:
Done